README.md

```diff
@@ -7,7 +6,11 @@
 
 HTTP request logger middleware for node.js
 
-> Named after [Dexter](http://en.wikipedia.org/wiki/Dexter_Morgan), a show you should not watch until completion.
+// I thought you meant: "The latter seasons of Dexter are no good and should be skipped"
```

Comment:
I'm not sure what all these added to the readme do. It seems to be javascript comments, but markdown just will render as plain text in the readme.